config/application.rb

```diff
@@ -14,14 +14,10 @@ class Application < Rails::Application
 
     # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
     # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
-    # config.time_zone = 'Central Time (US & Canada)'
+    config.time_zone = 'London'
```

Comment:
Not sure about this. I'd rather have it at UTC and handle timezone explicitly when needed. Thoughts?